extension/ggbn/permit_2022-08-08.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="/style/human.xsl"?>
+<extension xmlns="http://rs.gbif.org/extension/"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xmlns:dc="http://purl.org/dc/terms/"    
+    xmlns:dwc="http://rs.tdwg.org/dwc/terms/"    
+    xmlns:ggbn="http://data.ggbn.org/schemas/ggbn/terms/"    
+    xsi:schemaLocation="http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd"
+    dc:title="GGBN Permit Extension" 
+    name="Permit" namespace="http://data.ggbn.org/schemas/ggbn/terms/" rowType="http://data.ggbn.org/schemas/ggbn/terms/Permit"
+    dc:issued="2022-08-08"
+    dc:subject=""
+    dc:relation="http://terms.tdwg.org/wiki/GGBN_Permit_Vocabulary"
+    dc:description="Support for all kinds of permits as an extension for Material Sample core sample data in Darwin Core. Intended to be a one to many relation to the Material Sample core.">
+
+    <property group='Permit' thesaurus="http://rs.gbif.org/vocabulary/ggbn/permit_type.xml" name='permitType' namespace='http://data.ggbn.org/schemas/ggbn/terms/' qualName='http://data.ggbn.org/schemas/ggbn/terms/permitType' dc:relation='http://terms.tdwg.org/wiki/ggbn:permitType' dc:description='A permit is a document that allows someone to take an action that otherwise would not be allowed. Mandatory element' examples='     Collecting Permit, Import Permit, Export Permit, Intellectual Property Rights, Copyright, Patent, Data use, Phytosanitary, Salvage, Exemption Permit, Material Transfer Agreement, Internationally Recognized Certificate of Compliance, contract, Memorandum of Understanding, Memorandum of Cooperation, Veterinary Certificate, Human Pathogens, Genetically Modified Organism, Other' required='true'/>
```

Comment:
Should we surround examples with backticks? It'll be highlighted in the IPT then